Misc/NEWS.d/next/Library/2021-01-18-21-07-20.bpo-42960.a7Dote.rst

```diff
@@ -0,0 +1 @@
+Adds :data:`resources.RLIMIT_KQUEUES` constant from FreeBSD to the :mod:`resources` module
```

Comment:
please put a trailing dot to your sentence (and fix the typo in the module name)